doc/nrf/libraries/networking/nrf_cloud_pgps.rst

```diff
@@ -71,12 +69,10 @@ Configure these additional options to refine the behavior of P-GPS:
 * :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_FRAGMENT_SIZE`
 * :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_REQUEST_UPON_INIT`
 
-Configure the following option if you need your application to also use A-GPS, for coarse time and position data and to get the fastest TTFF:
-
-* :kconfig:option:`CONFIG_NRF_CLOUD_AGPS`
+Configure the :kconfig:option:`CONFIG_NRF_CLOUD_AGPS` option if you need your application to also use A-GPS, for coarse time and position data and to get the fastest TTFF.
```

Comment:
I would remove _coarse_, because I don't think that's really true. Sure, it's not as accurate as GNSS, but still. Actually, it makes more sense with the position, so maybe we should just move it. 😄 

We could also mention that A-GPS provides means to improve the accuracy.
```suggestion
Configure the :kconfig:option:`CONFIG_NRF_CLOUD_AGPS` option if you need your application to also use A-GPS, for time and coarse position data and to get the fastest TTFF.
Use of A-GPS also improves the accuracy because of ionospheric corrections.
```

Reply:
@peknis What about the new sentence I suggested for the accuracy improvement with A-GPS?

Reply:
Whoops, forgot that. Adding it now.